Drivers/STM32U5xx_HAL_Driver/Src/stm32u5xx_hal_rcc_ex.c

```diff
@@ -2902,7 +2903,7 @@ void HAL_RCCEx_EnableMSIPLLMode(void)
   */
 void HAL_RCCEx_DisableMSIPLLMode(void)
 {
-  CLEAR_BIT(RCC->CR, RCC_CR_MSIPLLEN);
+  MV_CLEAR_BIT(RCC->CR, RCC_CR_MSIPLLEN);
```

Comment:
Well, we certainly don't want userland ever clearing MSIPLLEN. What happens if this function is called by userland? does it fault?

Reply:
In HalSecurity.cpp we set MSISEC in RCC_SECCFGR, and since the mediated access version of the CR register has a write mask of zero, MV_CLEAR_BIT will end up writing to the non-secure mapping of this register.

According to RM0456 v0.5, section 11.5.1 a write to a secure bit is ignored (RAZWI)

Reply:
From a developer-experience point-of-view, if someone _thinks_ that they need to clear MSIPLLEN, what way of telling them that they can't wastes the least of their time? perhaps we should just comment out the whole function, or even poison the symbol?

Reply:
Well, this is the whole problem with RAZWI registers/bits, and the reason we had to implement MediatedAccess in the first place.

I think it warrants discussion, but isn't part of this ticket - commenting out the function or poisoning the symbol affects the microvisor porting experience in a bad way... RAZWI just means some cycles are wasted

Reply:
agreed, definitely not this ticket's problem, I've raised a story to trigger the discussion